translations/fr.yaml

```diff
@@ -229,7 +228,7 @@ general:
   selectVocabulary: "Choisi Vocabulaire"
   session: "Séance"
   sessionAdministration: "Administration de Séance"
-  sessionIs: "Cette séance est `{type}`"
+  sessionIs: "Type de Séance: `{type}`"
```

Comment:
Backticks should go away here in the same way quotes went away in the other translations.